composer.json

```diff
@@ -14,13 +14,12 @@
         "issues": "https://github.com/fastly/fastly-php/issues",
         "source": "https://github.com/fastly/fastly-php"
     },
-    "minimum-stability": "dev",
     "require": {
         "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "~6.0@dev"
+        "guzzlehttp/guzzle": "^6.2.0"
```

Comment:
@alieander 

There's probably no reason not to use the latest version of `guzzlehttp/guzzle` here, is there?